translations/base-pt-BR.yaml

```diff
@@ -1101,14 +1101,14 @@ keybindings:
         comparator: Comparador
         item_producer: Produtor de Itens (Sandbox)
         copyWireValue: "Fios: Copiar valor abaixo do cursor"
-        rotateToUp: "Rotate: Point Up"
-        rotateToDown: "Rotate: Point Down"
-        rotateToRight: "Rotate: Point Right"
-        rotateToLeft: "Rotate: Point Left"
-        constant_producer: Constant Producer
-        goal_acceptor: Goal Acceptor
-        block: Block
-        massSelectClear: Clear belts
+        rotateToUp: "Rotação: Para cima"
+        rotateToDown: "Rotação: Para baixo"
+        rotateToRight: "Rotação: Para direita"
+        rotateToLeft: "Rotação: Para esquerda"
+        constant_producer: Produtor Constante
+        goal_acceptor: Aceitador de Objetivo
```

Comment:
"Receptor de Objetivo" is probably a better name than "Aceitador de Objetivo" for the "goal_acceptor".